content/hi/docs/concepts/overview/_index.md

```diff
@@ -21,75 +21,75 @@ sitemap:
+* विकास, परीक्षण और उत्पादन में पर्यावरणीय संगतता: लैपटॉप पर वैसे ही चलता है जैसे क्लाउड में चलता है।
+* क्लाउड और ओएस (OS) वितरण पोर्टेबिलिटी: Ubuntu, RHEL, CoreOS, On-Premises, प्रमुख सार्वजनिक क्लाउड पर और अन्य कहीं भी चलता है।
+* एप्लिकेशन-केंद्रित प्रबंधन: वर्चुअल हार्डवेयर पर OS चलाने से लेकर तार्किक संसाधनों का उपयोग करके OS पर एप्लिकेशन चलाने तक अमूर्तता के स्तर को बढ़ाता है।
+* Loosely coupled, Distributed, Elastic-free, Microservices: एप्लिकेशन को छोटे, स्वतंत्र टुकड़ों में तोड़ा जाता है और उन्हें सक्रिय रूप से तैनात और प्रबंधित किया जा सकता है - एक बड़ी एकल-उद्देश्य वाली मशीन पर चलने वाला एक मोनोलिथिक स्टैक नहीं है।
 * संसाधन अलगाव: अनुमानित एप्लिकेशन प्रदर्शन।
 * संसाधन उपयोग: उच्च दक्षता और घनत्व।
 
 ## आपको कुबेरनेट्स की आवश्यकता क्यों है और यह क्या कर सकता है {#why-you-need-kubernetes-and-what-can-it-do}
 
-कंटेनर आपके एप्लिकेशनो को बंडल करने और चलाने का एक अच्छा तरीका है। उत्पादन के माहौल में, आप को उन कंटेनरों को प्रबंधित करने की आवश्यकता होती है जो एप्लिकेशने चलाते हैं और सुनिश्चित करते हैं कि कोई डाउनटाइम नहीं है। उदाहरण के लिए, यदि एक कंटेनर बंद हो जाता है, तो दूसरे कंटेनर को शुरू करने की आवश्यकता होती है। क्या यह आसान नहीं होगा यदि इस व्यवहार को एक प्रणाली द्वारा नियंत्रित किया जा सके ?
+कंटेनर आपके एप्लिकेशनों को बंडल करने और चलाने का एक अच्छा तरीका है। उत्पादन पर्यावरण में, आपको उन कंटेनरों को प्रबंधित करने की आवश्यकता होती है जो एप्लिकेशनों को चलाते हैं और सुनिश्चित करते हैं कि कोई डाउनटाइम नहीं है। उदाहरण के लिए, यदि एक कंटेनर बंद हो जाता है, तो दूसरे कंटेनर को शुरू करने की आवश्यकता होती है। क्या यह आसान नहीं होगा यदि इस व्यवहार को एक प्रणाली द्वारा नियंत्रित किया जा सके?
 
-इस तरह कुबेरनेट्स बचाव के लिए आता है! कुबेरनेट्स आपको वितरित सिस्टम को लचीलेपन से चलाने के लिए एक ढांचा प्रदान करता है। यह आपके एप्लिकेशन के लिए स्केलिंग और फेलओवर का ख्याल रखता है, डिप्लॉयमेंट पैटर्न प्रदान करता है, और भी बहुत कुछ। उदाहरण के लिए, कुबेरनेट्स आपके सिस्टम के लिए कैनरी  डिप्लॉयमेंट को आसानी से प्रबंधित कर सकता है।
+इस तरह कुबेरनेट्स बचाव के लिए आता है! कुबेरनेट्स आपको वितरित सिस्टम को लचीलेपन से चलाने के लिए एक ढांचा प्रदान करता है। यह आपके एप्लिकेशन के स्केलिंग और फेलओवर का ख्याल रखता है, डिप्लॉयमेंट पैटर्न के साथ और भी बहुत कुछ प्रदान करता है। उदाहरण के लिए, कुबेरनेट्स आपके सिस्टम के लिए कैनरी डिप्लॉयमेंट को आसानी से प्रबंधित कर सकता है।
 
 ## कुबेरनेट्स आपको प्रदान करता है:
 
 * **सेवा की खोज और लोड संतुलन**
-कुबेरनेट्स DNS नाम का उपयोग करके या अपने स्वयं के आईपी (IP) पते का उपयोग करके एक कंटेनर को उजागर कर सकता हैं। यदि एक कंटेनर में ट्रैफ़िक अधिक है, तो कुबेरनेट्स लोड बैलेंस करने और नेटवर्क ट्रैफ़िक को वितरित करने में सक्षम है ताकि डिप्लॉयमेंट स्थिर हो।
+कुबेरनेट्स DNS के नाम का उपयोग करके या अपने स्वयं के आईपी (IP) एड्रेस का उपयोग करके एक कंटेनर को प्रकट कर सकता है। यदि एक कंटेनर में ट्रैफ़िक अधिक है, तो कुबेरनेट्स लोड बैलेंस करने और नेटवर्क ट्रैफ़िक को वितरित करने में सक्षम है ताकि डिप्लॉयमेंट स्थिर हो।
```

Comment:
I don't think address needs to be transliterated. Rest of the changes, I agree with.
```suggestion
कुबेरनेट्स DNS के नाम का उपयोग करके या अपने स्वयं के आईपी (IP) पते का उपयोग करके एक कंटेनर को प्रकट कर सकता है। यदि एक कंटेनर में ट्रैफ़िक अधिक है, तो कुबेरनेट्स लोड बैलेंस करने और नेटवर्क ट्रैफ़िक को वितरित करने में सक्षम है ताकि डिप्लॉयमेंट स्थिर हो।
```